test/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# FILE(GLOB TestSources RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *_test.cpp)
+set(TestSources
```

Comment:
Is the glob cmmented because of broken tests and it will be back after we fix them?

Reply:
I was compiling only a few tests at a time, so I commented out glob so I could select which tests I wanted to compile. However, it's all good now, and the sources list has all of the test files.